cmd/destination/main.go

```diff
@@ -70,8 +73,32 @@ func main() {
 	// destination type coordinator which means every task with specific destination type
 	// will be published to its corresponding task publisher.
 
+	reconnectSecond := 5
+	channelSize := 100
+	numberInstant := 5
+	maxRetryPolicy := 5
```

Comment:
could you explain the retry policy in the destination I don't remember such a concept in destination

Reply:
It is used in the channel package. When a message is published to the input channel of the Rabbitmq adapter if it`s failed then retry until it is rich to the maxRetryPolicy.
We could add a delay time for retry in the future.

Reply:
thanks